src/org/openstreetmap/josm/plugins/continuosDownload/DownloadPreference.java

```diff
@@ -97,10 +97,8 @@ public void addGui(PreferenceTabbedPane gui) {
         panel.add(strategy, GBC.eol().fill(GridBagConstraints.HORIZONTAL).insets(5, 0, 0, 5));
 
         // quietDownload
-        quietDownload.setSelected(Config.getPref().getBoolean("plugin.continuos_download.quiet_download", false));
-        quietDownload.setToolTipText(tr("Suppress the progress monitor that is shown when downloading. If"
-                + " this option is selected there is no indication that something is being done, and no way to"
-                + " cancel the download."));
+        quietDownload.setSelected(Config.getPref().getBoolean("plugin.continuos_download.quiet_download", true));
+        quietDownload.setToolTipText(tr("Downloads the data continuously in the background as you pan and zoom around."));
```

Comment:
I'd really prefer to leave the warning `"If this option is selected there is no indication that something is being done, and no way to cancel the download."`.

Reply:
Sure. That could certainly be left in the tooltip. Better yet would be adding a download progress bar in the bottom row of the program, akin to the one in the old Internet Explorer.

When I tested, the error message that an area was a too big request for the server still came up when quiet_download was turned on, so maybe this isn't even an issue? How often would it be necessary to cancel the download rather than waiting a few seconds for the queue to clear up or for the server to throw an error?

Reply:
It kind of depends -- I'd much rather not depend upon the queue clearing up -- despite trying to do parallel downloads, the plugin actually does all the downloads in a single thread. I've fixed that in PR #8 , along with automatically stopping downloads when too many nodes are in the area.

The error will still occur with quiet download enabled, but the user might have done enough panning around that they get into what feels like an endless loop of closing those dialogs.

Reply:
In other words, let us keep the default at `false` for `plugin.continuos_download.quiet_download`, and after this PR and #8 are merged, wait a few months to see if anyone sees negative effects with `quiet_download` enabled. At that point, I'd be OK with enabling quiet_download by default.

Reply:
> The error will still occur with quiet download enabled, but the user might have done enough panning around that they get into what feels like an endless loop of closing those dialogs.
That's nothing new to JOSM, unfortunately. But should probably be avoided if possible :)

> In other words, let us keep the default at `false` for `plugin.continuos_download.quiet_download`, and after this PR and #8 are merged, wait a few months to see if anyone sees negative effects with `quiet_download` enabled. At that point, I'd be OK with enabling quiet_download by default.
Sounds good. 👍